.github/workflows/kind-e2e.yaml

```diff
@@ -50,6 +51,20 @@ jobs:
           - test-suite: ./test/e2e_channel
             extra-test-flags: -channels=messaging.knative.dev/v1beta1:KafkaChannel
 
+        exclude:
+          - test-suite: ./test/e2e
+            broker-class: KafkaNamespaced
+          - test-suite: ./test/e2e_sink
+            broker-class: KafkaNamespaced
```

Comment:
do we plan to, eventually, include those?

Reply:
Nah. The `exclude` here means excluding the combinations.
Like, "don't bother running KafkaSink E2E tests with broker class env var passed as `KafkaNamespaced`". Because we don't read that env var in sink e2e tests and simply create the regular broker. Or in the channel tests for example, we don't even create any broker.